traits/__init__.py

```diff
@@ -12,19 +15,6 @@
 # Add a NullHandler so 'traits' loggers don't complain when they get used.
 import logging
 
+logging.getLogger("traits").addHandler(logging.NullHandler())
```

Comment:
q : why use "traits" instead of `__name__`?

Reply:
More explicit; no need to think hard about whether `__name__` might ever possibly have the value `"__main__"`.

Reply:
Having said that, I don't feel strongly about the change, and the `__name__` version has greater copy-and-paste value.